packages/dm-core-plugins/src/attribute-selector/Icon.ts

```diff
@@ -0,0 +1,92 @@
+import { Icon } from '@equinor/eds-core-react'
+import {
```

Comment:
Could this possible be solved with a dynamic import? Making all EDS icons available without typing them out

Reply:
[will check](https://equinor.slack.com/archives/CJT20H1B9/p1681886587286679) before completing this PR :)